db/migrate/20230309191824_create_foods.rb

```diff
@@ -3,7 +3,8 @@ def change
     create_table :foods do |t|
       t.string :name
       t.string :measurement
-      t.integer :price
+      t.decimal :price
+      t.integer :quantity
       t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Kindly make sure to create the foods table before the `recipe_foods` so the migrations are created, as for now I can't migrate the db without reversing the order of the `foods` table and `recipe_foods` table this is because the second one depending on the foods table to be created first

Reply:
@AnselemOdims :wave: , We are not sure of what you meant by the above statements because clearly, the table for `foods` was created before that of `recipe_foods`. We would like if you can expatiate your viewpoint about this! Below is an attached screenshot of the present order of arrangement at creation:

![Screenshot from 2023-03-04 17-43-28](https://user-images.githubusercontent.com/99715640/222918239-e34d7d84-211e-427f-a1da-feacdf548db7.png)

Reply:
@AbayomiOlaoye, yes that is right but the migration files that you have now in your PR will not create the `schema` file instead it will return this error in the console instead👇🏼
![Screenshot_20230304_180909](https://user-images.githubusercontent.com/25227022/222919190-864429b6-5f32-43f5-a311-6bad716cad88.png)

I suspect that the `schema` file is generated by the previous code, is that right?

Reply:
That's strange, and no. It's a fresh migration and schema file. We had to drop the old dB before creating new controllers and schema.....Oh, I think I get you now. Yes, it's probably the old one 👍